src/fluid/fluid.cpp

```diff
@@ -207,6 +207,7 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   physics->AddField(p::entropy::name(), mprim_scalar);
   physics->AddField(p::cs::name(), mprim_scalar);
   physics->AddField(diag::ratio_divv_cs::name(), mprim_scalar);
+  physics->AddField(diag::entropy_z_0::name(), mprim_scalar);
```

Comment:
might make more sense to register this var in the analysis initialize function. But this is fine for now.